R/fancy_ideogram.R

```diff
@@ -85,7 +81,7 @@ fancy_ideogram = function(this_sample_id,
 
   #additional regions to plot
   if(!missing(gene_annotation)){
-    gene = gene_to_region(gene_symbol = gene_annotation, genome_build = "grch37", return_as = "df")
+    gene = GAMBLR.utils::gene_to_region(gene_symbol = gene_annotation, genome_build = "grch37", return_as = "df")
```

Comment:
There are maybe more examples like this, and I probably did this myself too - but looking now with fresh eyes wanted to get your thoughts on it. GAMBLR.utils relies on GAMBLR.data and here now the GAMBLR.data will rely on GAMBLR.utils, which sounds somewhat circular. We probably need some "ground 0" of the package that is not relying on other GAMBLR stuff, and I was having in mind that GAMBLR.data is the "ground 0". What do you think?